utils/dashboard-helper.ts

```diff
@@ -81,16 +97,10 @@ export const runHelper = async (
       }
       const responseJSON = await response.json();
 
-      const dashLines = JSON.stringify(responseJSON, null, 2).split('\n');
-
-      dashLines.forEach((line, lineNumber) => {
-        const output = checkLine(line);
-        if (output.length > 0) {
-          output.forEach((o) =>
-            warnings.push(`| ${o} | ${dash.filename} | ${lineNumber + 1} |`)
-          );
-        }
-      });
+      const output = getWarnings(responseJSON);
```

Comment:
[nit] I kind of expect what we get back from `getWarnings` to be called `warnings` and maybe how we process them later on `102` as being the formatted `output`